src/mmwave_node.cpp

```diff
@@ -158,11 +168,15 @@ class MMWaveNode : public rclcpp::Node {
 
           auto msg = empty_point_cloud(frame_id, num_objects);
           msg.header.stamp = now();
-          msg.data.resize(24 * num_objects);
-          for (int i = 0; i < num_objects; i++) {
-            memcpy(&msg.data[24 * i], &objects[i], 24);
-          }
 
+          // Filter points based on an snr_threshold, and populate
+          if(objects->snr > snr_threshold){
```

Comment:
Accepts/rejects entire point cloud based on SNR of the first point

Also, if the point cloud is empty, this is a buffer over-read